infrastructure/modules/serverless/cloudtrail.tf

```diff
@@ -126,14 +109,18 @@ resource "aws_kms_key" "this" {
 EOF
 }
 
+data "aws_kms_alias" "existing" {
+  name = "alias/${var.namespace}-kms-key"
+}
 
 resource "aws_kms_alias" "a" {
```

Comment:
We should give some meaning full name to this resource

Reply:
Added in 621f2fb80404ba20e621b516b17cd84b11fbf1bb